tests/test_fleet_inventory.py

```
@@ -0,0 +1,661 @@
+    def test_no_gap(self) -> None:
+        diff = diff_fleet(
+            expected=["Prekzursil/a"],
+            inventoried=["Prekzursil/a"],
+        )
+        self.assertEqual(diff.missing, [])
+        self.assertEqual(diff.dead, [])
+
+    def test_returns_frozen_dataclass(self) -> None:
+        diff = diff_fleet([], [])
+        self.assertIsInstance(diff, FleetDiff)
+        with self.assertRaises(Exception):
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)